### ISSUE TITLE
preload is not working with class based syntax 

### ISSUE THREAD

**Issue by PriyankaRPathak** [ConsultingMD/graphql-preload#36]
Hi,
I'm using rails 4.1.16 and ruby 2.3.8 in my application. Converting DSL to class based syntax cause issue as it throws below error.

preload is undefined.

Can anyone suggest alternate option?

**Comment by atatarenko** [comment]
I was able to continue using the gem by changing the way how it's included in the Gemfile, see https://github.com/ConsultingMD/graphql-preload/issues/33.

**Comment by PriyankaRPathak** [comment]
Have you found any straightforward documentation to convert DSL => class based syntax for each type?

**Comment by atatarenko** [comment]
Yeah, I found one, here it is – https://graphql-ruby.org/schema/class_based_api.html
